### PR TITLE
fix issues #53 and #61

### PR DESCRIPTION
**Pull request by fofaraz**

### CODE REVIEW

**Review by bakhshandeh — COMMENTED** [review]
Let's move `DISCOVERY_VALID_TIMESTAMP` to net.default.conf and set a 30 secs default for it.

`discovery_acceptable_delay`

**Review by bakhshandeh — CHANGES_REQUESTED** [review]
rename the route to `/ifconfig` as we mentioned on the issue. 
Its more geeky and professional.

**Review by bakhshandeh — CHANGES_REQUESTED** [review]
As we discussed the user should be able to set `PUBLIC_IP` in .env
If PUBLIC_IP is on the config, there is no need to call /ifconfig and get the ip.

**Comment by fofaraz** [comment]
changes applied
please review

File: src/utils/helpers.ts
Comment (fofaraz) [inline src/utils/helpers.ts:1]: test

File: src/utils/helpers.ts [hunk 1]
```diff
@@ -122,15 +123,32 @@ export function pub2json(pubkey: PublicKey, minimal: boolean=false): JsonPublicK
...
+    .then(checkValidIp)
+    .then(ip => ({success: true, ip}))
+    .catch(e => ({success: false, ip: null}));
+
+  if(muonIp.success)
+    return muonIp.ip;
+
   // @ts-ignore
-  let ip = await Promise.any([
+   let ip = await Promise.any([
```
[the remark this answers is not in the document]
Comment (fofaraz) [reply src/utils/helpers.ts:156]: This creates an issue. When running a devnet node on local machine, there are no other nodes on loca to call /ifconfig on them so ifconfig request fails.
in this case we can use defined PUBLIC_IP in .env 
Do we need to anything else for it?

**Comment by fofaraz** [comment]
Updated. Please review.

**Pull request merged**